docs/src/main/asciidoc/kafka-reactive-getting-started.adoc

```diff
@@ -534,21 +541,21 @@ First, compile both applications as native:
 
 [source, bash]
 ----
-mvn -f kafka-quickstart-producer package -Pnative -Dquarkus.native.container-build=true
-mvn -f kafka-quickstart-processor package -Pnative -Dquarkus.native.container-build=true
+mvn -f kafka-quickstart-producer package -Dnative -Dquarkus.native.container-build=true
+mvn -f kafka-quickstart-processor package -Dnative -Dquarkus.native.container-build=true
 ----
 
 Run the system with:
 
 [source, bash]
 ----
 export QUARKUS_MODE=native
-docker compose up --build
+docker-compose up --build
```

Comment:
Compose v2 will be part of the Docker CLI and available now.

Reply:
I thought there may be something I'm missing. Is that https://docs.docker.com/compose/cli-command/? I'm on Linux if that explains anything :-)

Reply:
I'm looking more into this and the only other place that uses `docker compose` (instead of `docker-compose`) is `amqp.adoc`. Similarly, in Quarkus quickstarts, the only other place that uses `docker compose` is `amqp-quickstart` (I've changed the Kafka quickstart to use `docker-compose` as well).

I think I'd personally prefer to stay on `docker-compose` for the time being, until v2 graduates out of beta.

Reply:
so, new docker versions provide `docker compose`, and `docker-compose` is getting retired. I guess we can still use `docker-compose` for now. So, for people not using the newest versions, it will work. At some point, we will have to change to `docker compose` (everywhere in the quarkus documentation).